### DISCUSSION TITLE
Banned from discord with no warnings before

### DISCUSSION THREAD

**Discussion by AhmedBonnah** [pterodactyl/panel#5203] [Support]
Hello, I have been banned from pterodactyl server for the first time with no warnings before 
The bot says i got "formal warning" However i got a ban out of this 
My last message was only "help" just to get the tail logs command , I never asked for support for unofficial discord script and i have been in this server for a long time asking for help for the "official installation" provided in Docs 
I would like to receive an unban or a proper explanation of what i think is a mistake 
Thank you for your help 

my discord id is 896089166738247721
_rin0

**Comment by danny6167** [comment]
Hi. It looks like you've been unbanned already.
Please let me know if that isn't the case.
